.circleci/config-approval.yml

```diff
@@ -91,7 +91,6 @@ jobs:
       - run:
           name: Running Tests
           command: dotnet test
-          working_directory: << pipeline.parameters.project_name >>
```

Comment:
Correct! The right config should be:

      - run:
          name: Running Tests
          command: dotnet test -c Release --logger "trx"

Reply:
Can you please change build command to `command: dotnet build -c Release`? and also removing working_directory like your change does.